specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-03-02-preview/managedClusters.json

```diff
@@ -532,6 +532,22 @@
               "$ref": "#/definitions/ManagedCluster"
             },
             "description": "The managed cluster to create or update."
+          },
+          {
```

Comment:
may define the following two header parameters globally and add reference to them here